Foundation/NSOperation.swift

```diff
@@ -99,7 +145,11 @@ public let NSOperationQueueDefaultMaxConcurrentOperationCount: Int = 0 // Unimpl
 public class NSOperationQueue : NSObject {
     
     public override init() {
-        NSUnimplemented()
+        maxConcurrentOperationCount = NSOperationQueueDefaultMaxConcurrentOperationCount
```

Comment:
To keep consistency within the source file, shouldn't we define this in the property itself? Also, the below assignment seems redundant since `suspend` is already initialized in the declaration.